Branch/BranchMainClass.m

```diff
@@ -280,7 +294,7 @@ - (void)urlAppleEvent:(NSAppleEventDescriptor *)event
 
 - (void)applicationDidFinishLaunchingNotification:(NSNotification*)notification {
     // TODO: Remove this?
-    BNCLogMethodName();
+    //BNCLogMethodName();
```

Comment:
Same as above re stubbing out debug code.

Reply:
I didn't have time to assess if this debug code would be worthwhile in the future.